byron/ledger/formal-spec/utxo.tex

```diff
@@ -209,22 +209,21 @@ \subsection{Witnesses}
     \begin{array}{r@{~\in~}lr}
       \fun{wits} & \Tx \to \powerset{(\VKey \times \Sig)}
       & \text{witnesses of a transaction}\\
-      \fun{hash_{spend}} & \Addr \mapsto \HashKey
-      & \text{hash of a spending key in an address}\\
+      \fun{hash_{vkey}} & \Addr \mapsto \HashKey
+      & \text{hash of a verifying key in an address}\\
     \end{array}
   \end{equation*}
   \caption{Definitions used in the UTxO transition system with witnesses}
   \label{fig:defs:utxow}
 \end{figure}
-
 \begin{figure}[htb]
   \begin{align*}
-    & \addr{}{} \in \UTxO \to \TxIn \mapsto \Addr & \text{address of an input}\\
+    & \addr{}{} \in \UTxO \to \big( \TxIn \mapsto \Addr \big) & \text{addresses of inputs}\\
```

Comment:
Maybe it'd suffice to clarify in the notation section that function signatures associate to the right, like in Haskell. I'd prefer this over explicit parenthesis. Otherwise we'd have to be consistent and put parenthesis in the other signatures as well ..

Reply:
I think the issue here was that it is easy not see the `\mapsto`, thinking it is just the normal implication arrow. But I agree we need to keep consistency with the entire spec.

Reply:
Oh, I see, now. Maybe remove the `\big` then? Or make the rest of the spec consistent? @Jimbo4350 thoughts?

Reply:
@dnadales I'll revert this